src/main/java/io/strimzi/utils/LogicalKafkaVersionEntity.java

```diff
@@ -0,0 +1,192 @@
+
+    private String jsonVersion;
+    private final List<LogicalKafkaVersion> logicalKafkaVersionEntities = new ArrayList<>();
+
+    public LogicalKafkaVersionEntity() {
+        // scrape json schema and fill the inner list of versions
+        this.resolveAndParse();
+    }
+
+    public static class LogicalKafkaVersion implements Comparable<LogicalKafkaVersion> {
+        private final String version;
+        private final String image;
+
+        public LogicalKafkaVersion(String version, String image) {
+            this.version = version;
+            this.image = image;
+        }
+
+        @Override
+        public int compareTo(LogicalKafkaVersion o) {
+            return compareVersions(this.version, o.version);
+        }
+
+        /**
+         * Compare two decimal version strings, e.g. 1.10.1 &gt; 1.9.2
+         * @param version1 The first version.
+         * @param version2 The second version.
+         * @return 0 if version1 == version2;
+         * -1 if version1 &lt; version2;
+         * 1 if version1 &gt; version2.
+         */
+        public static int compareVersions(String version1, String version2) {
+            String[] components = version1.split("\\.");
+            String[] otherComponents = version2.split("\\.");
+            for (int i = 0; i < Math.min(components.length, otherComponents.length); i++) {
+                int x = Integer.parseInt(components[i]);
+                int y = Integer.parseInt(otherComponents[i]);
+                // equality of version is ignored...
+                if (x < y) {
+                    return -1;
+                } else if (x > y) {
+                    return 1;
+                }
+
+            }
+            return components.length - otherComponents.length;
+        }
+
+        // f.e. "3.0.0"
+        public String getVersion() {
+            return version;
+        }
+
+        // f.e. "test-container:0.1.0-kafka-3.0.0
+        public String getImage() {
+            return image;
+        }
+
+        public String getStrimziTestContainerVersion() {
+            Matcher regex = STRIMZI_TEST_CONTAINER_IMAGE_WITHOUT_KAFKA_VERSION.matcher(this.image);
+
+            // only one occurrence in the string
+            if (regex.find()) {
+                final String strimziContainerVersion = regex.group(1);
+                LOGGER.info("Find the version of Strimzi kafka container:{}", strimziContainerVersion);
+                return strimziContainerVersion;
+            } else {
+                throw new RuntimeException("Unable to find version of Strimzi kafka container from image name - " + this.image);
+            }
+        }
+
+        @Override
+        public String toString() {
+            return "LogicalKafkaVersionEntity{" +
+                "version='" + this.version + '\'' +
+                ", image='" + this.image + '\'' +
+                '}';
+        }
+    }
+
+    /**
+     * Get the latest release
```

Comment:
Let's clarify what we mean by "latest" here, and explain how the result is intentionally not deterministic. It's the released `test-container-images` image with the highest Kafka version number. The result will change when a new version of Kafka is released with a higher Kafka version number, and a `test-container-images` image which contains that Kafka version has been built and released.